src/ActiveLogin.Authentication.BankId.AspNetCore.QRCoder/BankIdAuthenticationBuilderQrCoderExtensions.cs

```diff
@@ -0,0 +1,17 @@
+using ActiveLogin.Authentication.BankId.AspNetCore.Qr;
+using Microsoft.Extensions.DependencyInjection.Extensions;
+
+namespace ActiveLogin.Authentication.BankId.AspNetCore.QrCoder
+{
+    public static class BankIdAuthenticationBuilderQrCoderExtensions
+    {
+        public static IBankIdAuthenticationBuilder UseQrCoderQrCodeGenerator(this IBankIdAuthenticationBuilder builder)
+        {
+            var services = builder.AuthenticationBuilder.Services;
+
+            services.TryAddTransient<IBankIdQrCodeGenerator, BankIdQrCoderGenerator>();
```

Comment:
Tried this, but it does not override. You need to do: `AddTransient`

Reply:
Ah, of course. This has to override now.